nss-day-cohort-29/nutshell-midnight-dream
# Styling changes

# Description
Made a few styling changes. Current page is now highlighted in the nav bar.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout news-final-changes`
3. Run grunt
4. Click through all the navigation items and verify that the nav item changes to green after you click on it (and turns back to white when you click on another one).

# Checklist:
